app/views/admin/attachments/_reference_fields_legacy.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if attachable.respond_to?(:translated_locales) && attachable.translated_locales.many? %>
+  <%= form.label :locale, 'Display language' %>
+  <p class="help-block">This determines the translations of the publication that the attachment will appear in.</p>
+  <%= form.select :locale, options_for_locales(attachable.translated_locales), include_blank: 'All languages' %>
```

Comment:
This section needs to be updated to use the design systems select component

<img width="703" alt="image" src="https://user-images.githubusercontent.com/42515961/188887139-6ee9fb65-c4d0-40a7-b4ab-be1495e7b2b8.png">